### PR TITLE
Added functionality for new project creation.

### PR DESCRIPTION
**Pull request by slama6**

### CODE REVIEW

File: buildinglog/urls.py [hunk 1]
```diff
@@ -10,6 +10,6 @@
     url(r'^logout/$', auth_views.logout,
         {'next_page': '/login/'}, name='logout'),
     url(r'projects/$', views.projects.listview),
-    url(r'projects/new', views.projects.new_item),
+    url(r'projects/new_project', views.projects.new_item),
```
Comment (mskrip) [inline buildinglog/urls.py]: What is the reason behind this change?

Comment (slama6) [reply buildinglog/urls.py]: My first impression was that customers have in url new too... should I change it back?

Comment (mskrip) [reply buildinglog/urls.py]: Yes please, change it back. Having `projects/new` is enough for the context and having url as a context won't be necessary anyways after #36

Comment (slama6) [reply buildinglog/urls.py]: Fixed

**Comment by mskrip** [comment]
Please rebase from master. This PR does not have the right customer view implemented yet and would overwrite your previous changes in master.

**Comment by mskrip** [comment]
LGTM
